circle.yml

```diff
@@ -265,6 +268,39 @@ commands:
             echo $result
             if [ "$expected" != "$result" ]; then exit 1; fi
 
+  rust_restore_cargo_cache:
+    steps:
+      - restore_cache:
+          name: "Restore cargo cache"
+          key: cargo-
```

Comment:
Does this mean it will pick up anything? Latest?

Reply:
It suppose to download the latest matching this prefix. We cannot match with Cargo.lock because it does not exist here.